waiter/src/waiter/settings.clj

```diff
@@ -86,7 +86,8 @@
                                               s/Keyword schema/require-symbol-factory-fn}
                                              schema/contains-kind-sub-map?)
    ;; TODO port belongs in server-options?
-   (s/required-key :port) schema/positive-int
+   (s/required-key :port)  (s/either schema/positive-int
```

Comment:
Nitpick: Extra space between the closing and opening parenthesis.

Reply:
Done